### PR TITLE
Hm 06

### PR DESCRIPTION
**Pull request by zimniimars**

### CODE REVIEW

**Comment by zimniimars** [comment]
c

File: HW06_Task_2/HW06_Task_2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            int number = int.Parse(Console.ReadLine());
+            Insert(ref array, number, index2);
+            for (int i = 0; i < array.Length; i++)
+            {
+                Console.WriteLine($"Новый массив {i}:\t ");
+                Console.WriteLine(array[i]);
+            }
+        }
+    }
+}
```
Comment (VadzimPapko) [inline HW06_Task_2/HW06_Task_2/Program.cs:46]: correct

File: HW_06_Task1/Task1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] Aray1 = new int[5];
```
Comment (VadzimPapko) [inline HW_06_Task1/Task1/Program.cs:9]: variables should start from lower case

**Review by VadzimPapko — COMMENTED** [review]
Did not find Task 3. Your score is 7

**Pull request merged**